### PR TITLE
fix: update imports for SDK docs examples

### PR DESCRIPTION
**Pull request by fabiorigam**

### CODE REVIEW

File: developer-resources/sdks-and-providers/sdk/provider.md
Comment (nwbrettski) [inline developer-resources/sdks-and-providers/sdk/provider.md:1]: Can we undo these changes. I changed them manually on the last PR. The reason for the manual changes were due to naming conventions.

Comment (fabiorigam) [reply developer-resources/sdks-and-providers/sdk/provider.md:1]: It's better to overwrite instead. We need the SDK and the docs to be 100% aligned otherwise it's hard to understand which part to update. Also we should always change from SDK side, not docs side. We can automatize this process.

Comment (nwbrettski) [reply developer-resources/sdks-and-providers/sdk/provider.md:1]: Understood. I can raise these modifications as an issue on the board. Will resolve in this case.

**Review by nwbrettski — APPROVED** [review]
LGTM
